operation/operation.c

```diff
@@ -107,14 +107,22 @@ void logf_msg(enum logs logEnum, oper_st * state, int n_args, ...) {
             ;
     }
 
-    // LoRa messages to be sent to mqtt
+    // Cases where LoRa msg's will be sent, and how
```

Comment:
this description is very unclear, I would prefer the old description

Reply:
Fine. This pertains to clean up anyway.